python2/koans/about_control_statements.py

```diff
@@ -73,7 +73,7 @@ def test_for_statement_with_tuples(self):
             result.append("Contestant: '" + knight + \
             "'   Answer: '" + answer + "'")
 
-        text = __
+        text = "Contestant: 'Robin'   Answer: 'Blue! I mean Green!'"
```

Comment:
`for knight, answer in round_table`  is an example of unpacking. `for item in round_table`  would make `item` a tuple, but when writing `knight, answer` instead of `item`, Python will _unpack_ the tuple into its components.